pkg/ingress/controller/openstack/octavia.go

```diff
@@ -492,6 +492,11 @@ func (os *OpenStack) EnsurePoolMembers(deleted bool, poolName string, lbID strin
 		}
 		members = append(members, member)
 	}
+	// only allow >= 1 members or it will lead to openstack octavia issue
+	if len(members) == 0 {
```

Comment:
Why not fail immediately after querying all the nodes?

Reply:
because there are logic check above to avoid node failure case, 
so this check is to avoid either no nodes or has nodes but all nodes are failed

to be honest it seems openstack side change might be better (to avoid 500 error, return 400 bad request), but seems add this kind of 'no members' is a plus to have in provider side